packages/amplify-cli/src/commands/build-override.ts

```diff
@@ -23,6 +23,7 @@ export const run = async (context: $TSContext) => {
     if (categoryName && resourceName) {
       filteredResources = resourcesToBuild.filter(resource => resource.category === categoryName && resource.resourceName === resourceName);
     }
+    await context.amplify.invokePluginMethod(context, 'awscloudformation', undefined, 'transformResourceWithOverrides', [context]);
```

Comment:
What is the third param which is undefined here?

Reply:
this is service parameter, invoke plugin requires it in its implementation

Reply:
I think it's category and then resource-name which is expected. Just make a comment out here for maintainability as to why we pass in undefined in this case.